ngi_reports/utils/entities.py

```diff
@@ -71,9 +72,9 @@ def populate_sample(self, log, library_construction, **kwargs):
                 log.warning(
                     f"Prep status missing for sample {self.ngi_id} in prep {prep_id}"
                 )
-
+            for flowcell in prepObj.seq_fc:
+                self.flowcells.append(flowcell)
```

Comment:
Where is self.flowcells used? I mean the Sample class' one because we also have a similarly named one in Project